### PR TITLE
Single playlist video management

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Playlist Management

- I can navigate to my playlists where all my playlists are present with the names & videos.

On the Playlist Page,

- I can create a new playlist.
- I can delete an existing playlist
- I can remove a video from a particular playlist.

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
The code structure is good ols merge it.

File: src/context/servicesContext/servicesContext.jsx [hunk 1]
```diff
@@ -38,6 +39,21 @@ const ServiceProvider = ({ children }) => {
...
+  //       //   data: {
+  //       //     playlists: { videos }
+  //       //   }
+  //       // } = await getPlaylistOfUserService(authToken, playlistId);
+  //       // dispatch({ type: "MANAGE_SINGLE_PLAYLIST", payload: videos });
+  //     } catch (error) {
+  //       console.log("error in getting single playlist", error);
+  //     }
+  //   }
+
```
Comment (Tanay-J) [inline src/context/servicesContext/servicesContext.jsx]: do remove comments before merging

File: src/components/modals/playlist-modal/playlist-modal.module.css [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
   justify-content: center;
   align-items: center;
   position: absolute;
-  inset: 0;
-  background: var(--grey);
-  z-index: 1;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```
Comment (Tanay-J) [inline src/components/modals/playlist-modal/playlist-modal.module.css]: you may use `inset` shorthand property

**Review by Tanay-J — APPROVED** [review]
Good work! Liked the use of descriptive namin convention for classes and functions.
You may merge

**Pull request merged**